tests/countries/test_chile.py

```diff
@@ -270,12 +270,14 @@ def test_columbus_day(self):
         )
 
         self.assertHolidayName(
```

Comment:
No need to change this test, in #990 I've updated this holiday name, it was wrong :)

Reply:
I believe I'll have to resolve this when I merge it :)